frontend/device/sdl/device.lua

```diff
@@ -38,7 +38,7 @@ function Device:init()
         self.hasClipboard = yes
         self.screen = require("ffi/framebuffer_SDL2_0"):new{device = self, debug = logger.dbg}
 
-        local ok, re = pcall(self.screen.setWindowIcon, self.screen, "resources/koreader.png")
+        local ok, re = pcall(self.screen.setWindowIcon, self.screen, "resources/koreader-square.png")
```

Comment:
That's one of the things but there's also the icon.

Reply:
Yeah, I basically went with what was done in #3884, so I may have missed stuff ^^.

Reply:
Hmm, which icon? The one the desktop file points (or, well, failed to point to indeed ^^)?

Reply:
Yup. But how's this fmon stuff set up? I'm kind of thinking the only proper options for the "generic icon" would be either just the icon (with transparency) or the square version of it (with left/right padding).

Reply:
fmon picks up whatever you tell it to, which, given the current doc & Makefile, is what ends up as `koreader.png` (from `resources/koreader.png`).

And I totally agree that a generic icon should be square ;).